library/styleguideComponents/UniversalSearch/PostItemSearchResult.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LabelText from './labelText';
+import DescriptionText from './descriptionText';
+import {
+    ResultItemContainer,
+    IconContainer,
+    OptionText,
+    PostContainer,
+    StyledImage,
+    PostText,
+    Information,
+} from './styled';
+
+const ItemSearchResult = props => {
+    const { icon, title, description, image, date, author } = props;
+    const options = { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' };
+    const formattedPostDate = date
+        ? new Intl.DateTimeFormat(undefined, options).format(new Date(date))
+        : '';
+    console.log(image);
+    return (
+        <ResultItemContainer>
+            <IconContainer>{icon}</IconContainer>
+            <OptionText>
+                <LabelText value={title} />
+                <DescriptionText value={description} />
+                <PostContainer>
+                    <StyledImage url={image} alt="post" />
```

Comment:
since image is an URL and not an image itself we should change its name to imageUrl